fmn/rules/utils.py

```diff
@@ -162,6 +235,27 @@ def invalidate_cache_for(config, fn, arg):
     return _cache.delete(key)
 
 
+def _get_packages_for(config, username, flags):
```

Comment:
This seems very similar to `fmn.rules.utils. _get_packagers_for`. Is there a way to consolidate these?